grant.md

```diff
@@ -1,19 +1,245 @@
 ---
 title: GRANT
-toc: true
+toc: false
 ---
 
-## Description
+The `GRANT` [statement](sql-statements.html) grants privileges on one or more databases or tables to one or more users. 
+
+The user granting privileges must have the `GRANT` privilege on the target databases or tables.  
+
+<style>
+div#toc ul {
+    max-width: 50%;
+}
+</style>
+
+<div id="toc"></div>
 
 ## Synopsis
 
 {% include sql/diagrams/grant.html %}
 
-## Privileges
+## Supported Privileges
+
+CockroachDB supports the following privileges at the database and table levels. For details about privileges required by SQL statements not explicitly covered here, see the [SQL statement](sql-statements.html) documentation.
+
+- [`ALL`](#all)
+- [`CREATE`](#create)
+- [`DROP`](#drop)
+- [`GRANT`](#grant)
+- [`SELECT`](#select)
+- [`INSERT`](#insert)
+- [`DELETE`](#delete)
+- [`UPDATE`](#update)
+
+### `ALL`
+
+Level | Description
+------|------------
+Database | The user has all privileges on all tables in the database.
+Table | The user has all privileges on the specified table.
+
+### `CREATE`
+
+Note that only the `root` user can [create databases](create-database.html).
+
+Level | Description
+------|------------
+Database | The user can [create tables](create-table.html) and [indexes](create-index.html) in the database.
+Table | The user can [create indexes](create-index.html) on the specified table.
+
+### `DROP`
+
+Note that only the `root` user can [drop databases](drop-database.html).
+
+Level | Description
+------|------------
+Database | The user can [drop tables](drop-table.html) and [indexes](drop-index.html) in the database.
+Table | The user can [drop indexes](drop-index.html) on the specified table.
```

Comment:
removing indexes doesn't require `DROP`, it requires `CREATE` for schema modification.